text/0000-argument-spread-splarguments.md

```diff
@@ -0,0 +1,81 @@
+- Start Date: 2019-06-05
+- Relevant Team(s): Ember.js
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Forwarding Component Arguments with "Splarguments"
+
+## Summary
+
+Currently it is possible to forward element attributes and modifiers(?) into child components for use as "splattributes" 
```

Comment:
I feel like the wording here is a bit unclear. Maybe something along the lines of:

"Currently it is possible to forward element attributes and modifiers into child components using the spread-like `...attributes` syntax. This RFC proposes adding a similar syntax and functionality for component arguments: `...arguments`.

Reply:
Also, modifiers are forwarded by `...attributes`, so yeah we can remove the question mark there